template/FullStack/React(Frontend)+Nodejs(Backend)/docker-compose.yaml

```
@@ -0,0 +1,38 @@
+  
+  backend:
+    build:
+      context: ./server
+      dockerfile: Dockerfile
+    container_name: univ_template_nodejs_server
+    depends_on:
+      - mongodb
+    ports:
+      - "3000:3000"
+    environment:
+       MONGO_URL: mongodb://mongodb_uni:27017/uni
```

yamllint

[warning] wrong indentation: expected 6 but found 7

(indentation)